M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1614,50 +1784,124 @@ doc ///
         rank
 ///
 
-TEST /// --check #0 (Rn)
+doc ///
+    Key
+        isCodimAtLeast
+        (isCodimAtLeast, ZZ, Ideal)
+        [isCodimAtLeast, Verbose]
+        [isCodimAtLeast, SPairsFunction]
+        [isCodimAtLeast, PairLimit]
+        SPairsFunction
+    Headline
+        returns true if we can quickly see whether the codim is at least a given number
+    Usage
+        isCodimAtLeast(n, I)
+    Inputs
+        n: ZZ
+            an integer
+        I: Ideal
+            an ideal in a polynomial ring over a field, or a quotient ring
+    Outputs
+        : Boolean
+           {\tt true} if the codimension of I is at least n
+        : 
```

Comment:
This second output item conveys the idea that the function returns 2 outputs, rather than the idea that the function returns 1 output, which could be of two types.

Reply:
I see, I guess I don't know what the right way to document that is then.

Reply:
Try this:
```
    Outputs
        : Boolean
           {\tt true} or {\tt false}, indicating whether the codimension of I is at least n, if it can be determined, else {\tt null}
```

We cheat a little, in that "null" is not of type Boolean.

Reply:
Ok, I believe I've made this change throughout.